github/Rakefile

```diff
@@ -0,0 +1,84 @@
+require "yaml"
+require "json"
+require "octokit"
+
+def repos
+  @repos ||= YAML.load_file("data/repos.yml")
+end
+
+def client
+  Octokit.auto_paginate = true
+  @client ||= Octokit::Client.new(access_token: ENV.fetch("GITHUB_TOKEN"))
+end
+
+desc "Fetch the names of all GOV.UK repos from alphagov"
+task :fetch_repos do
+  repos = client.search_repositories("topic:govuk user:alphagov", per_page: 100).items.map(&:full_name)
```

Comment:
This will only expose public repositories. If you want to do private ones too you'll need to use `org_repos` and filter by topic.

Reply:
👍 updated.